src/main/java/ui/UI.java

```diff
@@ -174,6 +181,9 @@ private void showMainWindow(String repoId) {
         // Should only be called after panels have been initialized
         ensureSelectedPanelHasFocus();
         initialisePickers();
+        if (isFirstTimeUser && TestController.shouldOpenSampleBoard()){
+            openSampleBoard();
+        }
```

Comment:
Yup, this problem transcends beyond this PR. TestController is used throughout `UI.java`. Would have to create a separate issue for this. Will go ahead and do that.